src/run-single-fargate-task.py

```diff
@@ -192,6 +202,7 @@ def prepare_cmd(content, token):
             + token
             + ' --task-output \'{"output": "$result"}\' --region eu-west-1; else aws stepfunctions send-task-failure --task-token '
```

Comment:
Same here (eu-west-1 assumed) although I guess we didn't spot this on the last merge